packages/main/src/Input.hbs

```diff
@@ -25,7 +25,6 @@
 			aria-autocomplete="{{accInfo.input.ariaAutoComplete}}"
 			aria-expanded="{{accInfo.input.ariaExpanded}}"
 			aria-label="{{accInfo.input.ariaLabel}}"
-			aria-required="{{accInfo.input.ariaRequired}}"
```

Comment:
I understand the change, but just to note that if we remove that we can also remove the following as it is not used in the template anymore:
```js
// Input.js#1070
"ariaRequired": (this._inputAccInfo && this._inputAccInfo.ariaRequired) || this.required
```

And, second note - the DatePicker used to pass "accInfo.input.ariaRequired",
so it also should be removed if not necessary:

```js
// DatePicker.js#826
"ariaRequired": this.required,
```

But then I guess we have to forward the "required" property from the DatePicker to the Input in the DatePicker.hbs template. Otherwise, the Input (in the DatePicker) will remain without both aria-required and required, as it previously used only aria-required and now it is being removed from the Input.hbs template.

Reply:
We changed the approach. Now only "aria-required" is always added to the input of all input-based components. Also as you have mentioned, we forward the "required" property from the DatePicker to the Input.